### ISSUE TITLE
Doesn't support Meson

### ISSUE THREAD

**Issue by LDprg** [paldepind/projectdo#12]
Well meson is a widely used build system (mainly an alternative to cmake). It would be great if this supports meson!

**Comment by paldepind** [comment]
Hi @LDprg. Thanks for the suggestion. I think that is a great idea!

Are you interested in doing a PR? Otherwise I can take a look at it 🙂

**Comment by LDprg** [comment]
@paldepind maybe you should take a look at it.

**Comment by paldepind** [comment]
Sure, no problem.

Please see #13 which adds Meson support. It would be great if you could try it out and confirm that it works as it should. You can get the updated script from the PR with:

```sh
curl https://raw.githubusercontent.com/paldepind/projectdo/support-meson/projectdo -o ~/bin/projectdo
chmod +x ~/bin/projectdo
```

where `~/bin/` is any directory in your path :)

**Comment by paldepind** [comment]
Closed by #13.
